index.html

```diff
@@ -68,7 +54,10 @@ <h1>
           </div>
 
           <div class="line_3d">
-            <img src="img/3d_line.png" alt="">
+            <picture>
+              <source srcset="img/3d_line.webp" type="image/webp">
+              <img src="img/3d_line.png" alt="">
+            </picture>
```

Comment:
It's ok to remove the "png" images completely. [WebP is supported by all major browsers](https://caniuse.com/?search=webp) and is ok to use as the only option.

<img width="1269" alt="image" src="https://github.com/user-attachments/assets/d4ef62b9-2ba6-4c7a-a146-89fa39a97fb8">